sympy/physics/vector/frame.py

```diff
@@ -634,17 +633,12 @@ def orientnew(self, newname, rot_type, amounts, rot_order='', variables=None,
         >>> N = ReferenceFrame('N')
         >>> A = N.orientnew('A', 'Axis', [q1, N.x])
 
-
-        .orient() documentation:\n
-        ========================
-
         """
 
         newframe = self.__class__(newname, variables, indices, latexs)
         newframe.orient(self, rot_type, amounts, rot_order)
         return newframe
 
-    orientnew.__doc__ += orient.__doc__
```

Comment:
If you are going to remove this, you should add in a full explanation of all the options to `orientnew` here.